cmd/init.go

```diff
@@ -18,14 +18,19 @@ func init() {
 var initCmd = &cobra.Command{
 	Use:   "init [project]",
 	Short: "Initialize a flutter project to use go-flutter",
-	Args: func(cmd *cobra.Command, args []string) error {
-		if len(args) != 1 {
-			return errors.New("requires one argument, the project path. e.g.: github.com/my-organization/my-app")
-		}
-		return nil
-	},
 	Run: func(cmd *cobra.Command, args []string) {
-		projectPath := args[0]
+		var projectPath string
+		if len(args) == 0 || args[0] == "." {
+			projectName := getPubSpec().Name
```

Comment:
What if theres no pubspec? Probably `assertInFlutterProject()` should happen before this.